### ISSUE TITLE
BranchUniversalObject::showShareSheet displays a mostly empty full-height sheet

### ISSUE THREAD

**Issue by ddekanski** [BranchMetrics/android-branch-deep-linking#619]
Invoking showShareSheet with the default ShareSheetStyle of `new ShareSheetStyle(context, title, description)` displays an ugly full-height list as presented at the bottom of this post.

A possible culprit is the `AnimatedDialog::setDialogWindowAttributes()` method:

```    
public void setDialogWindowAttributes() {
        requestWindowFeature(Window.FEATURE_NO_TITLE);
        getWindow().setBackgroundDrawable(new ColorDrawable(Color.TRANSPARENT));
        getWindow().addFlags(WindowManager.LayoutParams.FLAG_DIM_BEHIND);
        getWindow().addFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN);
        WindowManager.LayoutParams lp = new WindowManager.LayoutParams();
        lp.copyFrom(getWindow().getAttributes());
        lp.width = WindowManager.LayoutParams.MATCH_PARENT;
        lp.height = WindowManager.LayoutParams.MATCH_PARENT;
        lp.gravity = Gravity.BOTTOM;
        lp.dimAmount = 0.8f;
        getWindow().setAttributes(lp);
        getWindow().setWindowAnimations(android.R.anim.slide_in_left);
        setCanceledOnTouchOutside(true);
    }
```
![image](https://user-images.githubusercontent.com/7956364/46367391-b4b87800-c67d-11e8-8e72-22b82f3effd2.png)

**Comment by parthnumbertank** [comment]
Any update?

**Comment by appvideocontact** [comment]
face the same problem. Any update?

**Comment by Sarkar** [comment]
Hi all, sorry for the delay. We're going to get this tackled in the coming week or so. To confirm, @parthnumbertank, @appvideocontact, is this only on Samsung phones, or are you seeing this on others?

**Comment by ddekanski** [comment]
@Sarkar I experienced it only on my Samsung S7 Edge, the issue doesn't appear on my other phones nor in an emulator. But still, Samsung devices make up most of Android smartphones used globally.

**Comment by appvideocontact** [comment]
@Sarkar Yes, only on Samsung phones.

**Comment by ryankanno** [comment]
@Sarkar I'm seeing this on Pixel 2 / Pixel 2 XL.

**Comment by felipejm** [comment]
I think I found the problem.

When i test in a Samsung phone, the height of the OneDrive app icon is too big. It has 761px, while the others apps icons have 100px to 190px.

You can test verify this by debugging the variable viewItemMinHeight_ inside ShareItemView class.

**Comment by bhagyae5308** [comment]
> I think I found the problem.
> 
> When i test in a Samsung phone, the height of the OneDrive app icon is too big. It has 761px, while the others apps icons have 100px to 190px.
> 
> You can verify this by debugging the variable viewItemMinHeight_ inside ShareItemView class.

Yeah, I just verified in Samsung S6 device. After removing OneDrive app the overlay is coming properly.
@Sarkar When can we expect the fix regarding this from Branch.io

**Comment by bhagyae5308** [comment]
> @bhagyae5308 we just merged this in: #643
> 
> This should be going out in the next deploy, which should happen by mid next week.

Ok, thanks.

**Comment by csalmi-branch** [comment]
Since this issue has been fixed in a previous deploy, I am closing this now. If you are encountering this problem still, please reach out to our support team (support.branch.io) or open a new github issue and we would be happy to help you out!
